src/components/pages/Reservations.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */
+import './../pages/styles/reservations.css'
+
+import React, { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux';
+import { deleteReservation } from '../../redux/apiCalls';
+
+const Reservations = () => {

```

Comment:
- [ ] When I add a reservation, the reservation seems to be created successfully.  However, when I go to "My reservations" to see it, the app encounters an error and nothing is shown:

![image](https://user-images.githubusercontent.com/57421823/227381557-ae2539ae-300e-4cd7-aa3a-5f714e7868e1.png)

![image](https://user-images.githubusercontent.com/57421823/227381641-0039a755-1ba2-4325-8770-d9dc873a3cd0.png)



If this is a bug, kindly fix it so your app runs as intended. 👍